survey/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,119 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I assume the easiest thing here will just be to have someone put the translations directly in this .po file?